seacrowd/sea_datasets/up2/up2.py

```diff
@@ -0,0 +1,203 @@
+"""
+
+_HOMEPAGE = "https://universalpropositions.github.io/"
+
+_LANGUAGES = ["ind", "vie"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.CDLA_SHARING_1_0.value
+
+_LOCAL = False
+
+_URLS = {
+    split: {
+        "ind": [
+            f"https://raw.githubusercontent.com/UniversalPropositions/UP_Indonesian-GSD/main/id_gsd-up-{split}.conllup",
+            f"https://raw.githubusercontent.com/UniversalDependencies/UD_Indonesian-GSD/master/id_gsd-ud-{split}.conllu",
+            # f"https://raw.githubusercontent.com/indolem/indolem/main/dependency_parsing/UD_Indonesian_GSD/id_gsd-ud-{split}.conllu",   # there are missing sent_id from the IndoLEM's dataset.
+        ],
+        "vie": [
+            f"https://raw.githubusercontent.com/UniversalPropositions/UP_Vietnamese-VTB/main/vi_vtb-up-{split}.conllup",
+            # f"https://raw.githubusercontent.com/UniversalDependencies/UD_Vietnamese-VTB/master/vi_vtb-ud-{split}.conllu", # new data => mismatch.
+            f"https://raw.githubusercontent.com/UniversalDependencies/UD_Vietnamese-VTB/0edef6d63df949aea0494c6d4ff4f91bb1959019/vi_vtb-ud-{split}.conllu",  # r2.8
+        ]
+    }
+    for split in ["train", "test", "dev"]
+}
+
+# TODO: add supported task by dataset. One dataset may support multiple tasks --> # TODO: add supported task by dataset. One dataset may support multiple tasks.
+_SUPPORTED_TASKS = []
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class UP2Dataset(datasets.GeneratorBasedBuilder):
+    """
+    Southeast Asian language subsets from Universal Propositions (UP) 2.0 dataset.
+    Semantic role labeling (SRL) is a shallow semantic parsing task that identifies “who did what to whom when, where etc” for each predicate in a sentence.
+    It provides an intermediate (shallow) level of a semantic representation that helps the map from syntactic parse structures to more fully-specified representations of meaning.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        *[SEACrowdConfig(
+            name=f"{_DATASETNAME}{'_' if _LANG else ''}{_LANG}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}{'_' if _LANG else ''}{_LANG}",
+        ) for _LANG in ['', *_LANGUAGES]],
+        # *[SEACrowdConfig(
+        #     name=f"{_DATASETNAME}{'_' if _LANG else ''}{_LANG}_seacrowd_[seacrowd_schema_name]",
+        #     version=datasets.Version(_SEACROWD_VERSION),
+        #     description=f"{_DATASETNAME} SEACrowd schema",
+        #     schema="seacrowd_[seacrowd_schema_name]",
+        #     subset_id=f"{_DATASETNAME}{'_' if _LANG else ''}{_LANG}",
+        # ) for _LANG in ['', *_LANGUAGES]],
```

Comment:
I assume this one doesn't have a SEACrowd schema. I guess we can delete this commented block?

Reply:
done. 👍